source/extensions/access_loggers/grpc/http_grpc_access_log_impl.cc

```diff
@@ -140,10 +140,9 @@ void HttpGrpcAccessLog::emitLog(const Http::RequestHeaderMap& request_headers,
 
     for (const auto& header : response_headers_to_log_) {
       const auto entry = response_headers.get(header);
-      if (!entry.empty()) {
-        // TODO(https://github.com/envoyproxy/envoy/issues/13454): Potentially log all header
-        // values.
-        logged_headers->insert({header.get(), std::string(entry[0]->value().getStringView())});
+      const auto all_values = Http::HeaderUtility::getAllOfHeaderAsString(response_headers, header);
+      if (all_values.result().has_value()) {
+        logged_headers->insert({header.get(), std::string(all_values.result().value_or(""))});
```

Comment:
Because we has ensured tht the `result()` has a value so there is unnecessary to use `value_or`.

Reply:
sure should use `value` now

Reply:
done